web/static/js/components/idea_submission_form.jsx

```diff
@@ -26,26 +27,37 @@ class IdeaSubmissionForm extends Component {
     this.setState({ category: event.target.value })
   }
 
+  handleToggleShowCategories(event) {
+    this.setState({ showCategories: !this.state.showCategories })
+  }
+
+  handleToggleChange(event) {
+    this.props.onToggleActionItem()
+    this.handleToggleShowCategories()
+    this.setState({ category: "action-item"})
```

Comment:
Pulled the branch and that's indeed the behavior, but I realize in retrospect that going back to the previously selected category wasn't in the acceptance criteria, so it doesn't block the PR. continuing in review